tests/test_builder.py

```diff
@@ -75,11 +75,11 @@ def test_invalid_soft_delete_keys():
     with pytest.raises(InvalidConfigurationException) as excinfo:
         SchemaBuilder.validate_schema_config(config)
 
-    assert "The SOFT_DELETE key in RAW_SCHEMA_1 must map" in str(excinfo.value)
+    assert "The SOFT_DELETE key in DB_1.RAW_SCHEMA_1 must map" in str(excinfo.value)
 
     config = {
-        'APP_1': {
-            'RAW_SCHEMA_2': {
+        'DB_1.APP_1': {
+            'DB_1.RAW_SCHEMA_2': {
```

Comment:
Perhaps I'm missing something, but is there a test for happy-path where the source and destination schemas are different?  Is that necessary/useful to test?

Reply:
yes, absolutely. I think I was doing that in my head. I'll add it

Reply:
actually, it's a bit tougher than I remembered to be able to test that. After some consideration, I don't think it is as useful as I thought, as using a source and destination is more of a DBT/Snowflake role issue and less of something that this code is actually doing.